subsys/bluetooth/services/dis.c

```diff
@@ -38,6 +47,46 @@ static struct dis_pnp dis_pnp_id = {
 };
 #endif
 
+#if defined(CONFIG_BT_GATT_DIS_SETTINGS)
+#define DIS_STR_MAX
+static u8_t dis_model[CONFIG_BT_GATT_DIS_STR_MAX] = CONFIG_BT_GATT_DIS_MODEL;
```

Comment:
All these can be `const`.

Reply:
It is not consts and also you cannot take the address of a const in all systems.